subsys/dfu/dfu_target/src/dfu_target_full_modem.c

```diff
@@ -60,17 +59,27 @@ int dfu_target_full_modem_init(size_t file_size, int img_num,
 
 int dfu_target_full_modem_offset_get(size_t *out)
 {
+	if (!configured) {
```

Comment:
Will this function be called from `dfu_target.c` in the case where its not `configured`?

Reply:
Not true. The `dfu_target_offset_get()` is a public API, but it can only be called after `dfu_target_init()` in which case the full_modem target is already configured.
The full modem init() similarly fails if it has not been configured first.